priam/src/main/java/com/netflix/priam/backup/BackupVerification.java

```diff
@@ -108,6 +108,43 @@ private IMetaProxy getMetaProxy(BackupVersion backupVersion) {
         return Optional.empty();
     }
 
+    public List<BackupVerificationResult> verifyAllBackups(
+            BackupVersion backupVersion, boolean force, DateRange dateRange)
```

Comment:
Where are we using this flag of `force` here? I thought force means we re-verify the backups. If it is not used here better to remove this flag.

Reply:
Sure I'll remove it.